src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/SpotlightBatchController.java

```diff
@@ -165,7 +154,7 @@ public ResponseEntity<String> sendQueuedBatchesAndProcessSpotlightResponse() {
         return ResponseEntity.ok().body("Success");
     }
 
-    @GetMapping("/get-spotlight-scheme-errors/{schemeId}")
+    @GetMapping("/scheme/{schemeId}/spotlight/get-errors")
```

Comment:
This also isn't RESTful. 

In general if you have a `/` it should be followed by some sort of identifier and it should be hierarchical, so ideally each `/` takes you one level lower in the object tree. I understand that this isn't always possible but if we could stick to that as closely as possible it will keep things clean. 

REST API URI names should also not contain verbs such as GET, POST, PUT, DELETE.

A more appropriate name might be something like `@GetMapping("/scheme/{schemeId}/spotlight-errors")`